README.md

```diff
@@ -1,10 +1,13 @@
 WebMock fixtures
 ================
+[![Gem Version](https://badge.fury.io/rb/webmock-fixtures.svg)](https://rubygems.org/gems/webmock-fixtures)
+[![Build Status](https://travis-ci.org/underdogio/webmock-fixtures.svg)](https://travis-ci.org/underdogio/webmock-fixtures)
```

Comment:
We are going to want to use the `master` branch's status. Otherwise, this will show status for the latest build regardless of branch (e.g. `dev/*`, PRs):

```md
[![Build Status](https://travis-ci.org/underdogio/webmock-fixtures.svg?branch=master)](https://travis-ci.org/underdogio/webmock-fixtures)
```